lib/src/json_schema/json_schema.dart

```diff
@@ -497,6 +587,42 @@ class JsonSchema {
   /// Default: [JsonSchemaVersions.draft6]
   String get schemaVersion => _root._schemaVersion ?? JsonSchemaVersions.draft6;
 
+  /// Remote [Uri] the [JsonSchema] was fetched from, if any.
+  Uri get fetchedFromUri => _fetchedFromUri;
```

Comment:
All these new getters should all be private until we prove consumers need access, otherwise this is dangerous API surface to expose.